runtime/rasdump/javadump.cpp

```diff
@@ -4643,6 +4643,23 @@ JavaCoreDumpWriter::writeThreadBlockers(J9VMThread* vmThread, UDATA vmstate, j9o
 			// Should only occur for WAITING threads.
 			// (For BLOCKED threads this would indicate a broken VM)
 			_OutputStream.writeCharacters("<unowned>");
+
+			// Get J9VMThread details from initializeStatus field.
+			J9Class *lockClass = J9VMJAVALANGJ9VMINTERNALSCLASSINITIALIZATIONLOCK_OR_NULL(vmThread->javaVM);
+			if (NULL != lockClass) {
+				J9Class *clazz = J9OBJECT_CLAZZ(vmThread, lockObject);
```

Comment:
I think `lockObject` may be `NULL` here; please check for that.

Reply:
Thanks, Added condition to check that.